### ISSUE TITLE
Add accessible view for debug console

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#213413]
Requested by @jooyoungeo

This would just be a flat view of the whole debug console's contents similar to the terminal accessible view

**Comment by meganrogge** [comment]
@jooyoungseo, I just tested the current experience. I can see why in some cases, you want to inspect content in the accessible view. 

However, I worry that dumping the fully expanded content into the accessible view would be confusing and difficult to navigate. 

Can you please provide an example of an object that is currently problematic to interact with in the debug console?

**Comment by meganrogge** [comment]
Just went down that rabbit hole again and the issue is we're using the `contextViewService` for this, and that `render` call cannot be async

**Comment by meganrogge** [comment]
I also notice there's an icon that says `Click to expand` which is currently not keyboard accessible

**Comment by meganrogge** [comment]
I figured out a way to resolve the child nodes and update the accessible view's content.
